gradle.properties

```diff
@@ -17,3 +17,5 @@ POM_SCM_DEV_CONNECTION=scm:git:ssh://git@github.com:BranchMetrics/Segment-Branch
 POM_LICENCE_NAME=The MIT License (MIT)
 POM_LICENCE_URL=http://opensource.org/licenses/MIT
 POM_LICENCE_DIST=repo
+POM_DEVELOPER_ID=branch
```

Comment:
Won't we need to bump the version number in the gradle properties as well? I mean, we can do that as a separate release, but it might be better to have it here if we are just going to release all of these changes as soon as this is merged.

Reply:
The dev ID and dev Name  were missing in the staging and it causes compilation issue.
Yes. Version number needed to be bumped up before release  but do we need it to add it feature PR or to master at time of release as mentioned in Readme? ( My suggestion is to add on master at the time of release assuming there will be more features)